.github/workflows/conventional-commits.yml

```diff
@@ -0,0 +1,19 @@
+name: Conventional Commits
+
+on:
+  pull_request:
+    branches:
+      - master
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is it needed?

Reply:
@Youssef1313 This is only necessary for private repos. But we use it on our other repos like this.

Reply:
I think we can remove it then if not necessary for public repos